go.mod

```
@@ -39,7 +41,7 @@ require github.com/syndtr/goleveldb v1.0.1-0.20210819022825-2ae1ddf74ef7
 
 require (
 	github.com/Masterminds/semver/v3 v3.3.0
-	github.com/cometbft/cometbft-db v0.14.1
+	github.com/cometbft/cometbft-db v1.0.1
```

I don't think it's okay to bump a major version here

```suggestion
	github.com/cometbft/cometbft-db v0.15.0
```